mhkit/dolfyn/io/nortek_defs.py

```diff
@@ -340,3 +340,144 @@ def sci_func(self, data):
                     standard_name='signal_intensity_from_multibeam_acoustic_doppler_velocity_sensor_in_sea_water',
                     ),
 }
+
+waves_hdrdata = {
+    'time_alt': _VarAtts(dims=[],
+                         dtype=np.float64,
+                         group='coords',
+                         default_val=nan,
+                         units='seconds since 1970-01-01 00:00:00 UTC',
+                         long_name='Time',
+                         standard_name='time',
+                         ),
+    'batt_alt': _VarAtts(dims=[],
+                         dtype=np.uint16,
+                         group='data_vars',
+                         default_val=nan,
+                         factor=0.1,
+                         units='V',
+                         long_name='Battery Voltage',
+                         ),
```

Comment:
this one doesn't have a `standard_name`

Reply:
Variables don't get standard names if they aren't listed in this table: https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html